dwa_local_planner/src/dwa_planner.cpp

```diff
@@ -85,6 +85,8 @@ namespace dwa_local_planner {
     backwardvel_scale_ = 1.2 * config.sim_time * (config.path_distance_bias + config.goal_distance_bias);
     prefer_forward_costs_.setScale(backwardvel_scale_);
 
+    max_backward_sq_dist_ = std::pow(config.max_backward_dist, 2);
```

Comment:
```suggestion
    max_backward_sq_dist_ = config.max_backward_dist * config.max_backward_dist;
```
S: keep style consistent with rest of ros-navigation repo